AdminDashboard/BusinessLogicOrchestrators/AccountOrchestrator/AccountOrchestrator.cs

```diff
@@ -43,9 +43,11 @@ public async Task UpgradePlan(Guid accountId, UpgradePlanRequest request)
 
             if (DoesntExist(account))
                 throw new AccountNotFoundException();
+
             if (!IsPlanValid(request.Plan))
                 throw new InvalidPlanException();
-            if (!IsProposedPlanHigherThanCurrentPlan(request.Plan, account))
+
+            if (!IsProposedPlanLessThanCurrentPlan(request.Plan, account))
```

Comment:
Ha, I didn't even catch this the first time around. I was referring to the fact that you could just remove the `!` negation here and change the function. Either way, this is fine :)

Reply:
Aha! Changing.